adal/build.gradle

```diff
@@ -229,9 +229,37 @@ publishing {
         }
     }
 
+    // Repositories to which Gradle can publish artifacts
+    if (System.getenv("VSTS_ENV_ACCESS_TOKEN") != null || project.hasProperty("vstsMavenAccessToken")) {
+        repositories {
+            maven {
+                name "vsts-maven-adal-android"
+                url "https://identitydivision.pkgs.visualstudio.com/_packaging/AndroidADAL/maven/v1"
+                credentials {
+                    username "VSTS"
```

Comment:
can we move the url and username to environment variables as well, don't complicate the null check above, you can assume that if the access token is there then the other variables are there as well

Reply:
I could do that. Just one thing, the VSTS access token is set on each server, while the url is set for each repo. If we need to do the RC build for other repos in the future, we need to send the url for them as well.

Reply:
A URL for a Maven repository is required in VSTS build. So we could not set the url into the environment variable. It failed during debugging the publish task.

Reply:
@addev-ashish could you approve this PR if it LGTU? Thanks.